docker/local/docker-compose.yml

```diff
@@ -1,5 +1,18 @@
 version: "3.2"
 services:
+    dind:
```

Comment:
I don't suppose there is an easy way to make this conditional on whether docker is enabled?

Reply:
The docker-compose yaml specification does not have options to define optional services. I thought about this a bit and what we could do is split it up in 2 docker-compose yaml files? One is the main one without any dind container and a seperate one for if you want to run dependendecies in containers. wdyt?
So you run an extra `docker-compose -f docker-compose-dind.yaml` up when you need it.

Reply:
Sure, yeah that sounds good.  I'll defer to you on whether you want to do that in this PR or just get this one in and do it in a separate one.